content/rdi/installation/adding-tables-to-existing-pipeline.md

```diff
@@ -114,14 +114,74 @@ The data-collections array lists tables by their fully-qualified names, using th
 
 #### Signaling Table Columns
```

Comment:
In general, I think Redis' convention for headers is like this:
`Signalling table columns`.

@andy-stark-redis please, confirm. 
In this case, we should apply to all headers and sub-headers in this doc.